src/hotspot/share/memory/allocation.cpp

```diff
@@ -111,7 +111,7 @@ void* ArenaObj::operator new(size_t size, Arena *arena) throw() {
 // AnyObj
 //
 
-void* AnyObj::operator new(size_t size, Arena *arena) throw() {
+void* AnyObj::operator new(size_t size, Arena *arena) {
```

Comment:
Is this function guaranteed to crash in case of allocation failure? @afshin-zafari removed `throw()` in the header file in 0f51e6326373ff7d4a4d9a0e3a2788401f73405d, but left the one here... should we restore the `throw()` in the header file instead?

Reply:
[not a review, just a drive-by comment]
arena->Amalloc takes a second argument that defaults to EXIT_OOM, so this function does not return nullptr.
The earlier removal of the nothrow specification from the header was correct, but this one got missed, and
should be removed too.  So the change here is correct.